frontend/src/metabase/admin/settings/auth/components/AuthCard/AuthCard.styled.tsx

```diff
@@ -36,9 +36,12 @@ interface CardBadgeProps {
 }
 
 export const CardBadge = styled.div<CardBadgeProps>`
-  color: ${props => color(props.isEnabled ? "brand" : "danger")};
+  color: ${props =>
+    props.isEnabled ? "var(--mb-color-brand)" : color("danger")};
```

Comment:
note to other reviewer: danger is addressed in part 4, see https://github.com/metabase/metabase/pull/43608